integration/run_test.go

```diff
@@ -224,8 +225,9 @@ func TestKritisISPLogic(t *testing.T) {
 					name: "nginx-no-digest",
 				},
 			},
-			shouldSucceed: false,
-			dir:           "../",
+			shouldSucceed:  false,
+			attestedImages: []string{},
+			dir:            "../",
```

Comment:
Heads up: The dir attribute will be removed by https://github.com/grafeas/kritis/pull/237

Reply:
thanks. will get my PR in first :)